packages/libsolv/libsolv.spec

```diff
@@ -1,60 +1,59 @@
-%global python3_pkgversion 3.11
```

Comment:
Do we really need to drop the python 3.11 bindings?

Reply:
we probably want to _keep_ it even :D  Hence why I [wip] 'ed it

I did the initial copy over, but have not had a chance to update it to also build for python 3.11